### PR TITLE
ChangePasswordForm + UserMenu

### PR DESCRIPTION
**Pull request by mattcorner**
New components
* **ChangePasswordForm**
  Allows user to set a new password (with retype password) and submit form. Intended for user in an authenticated change password dialog, and an unauthenticated change password form (with reset token via email).
* **UserMenu**
  New user menu component for reuse across the apps. Changes from our current are to display username, and initials to quickly identify the logged in user.

### CODE REVIEW

File: src/UserMenu/UserMenu.js [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+  Divider,
+  IconButton,
+  makeStyles,
+  ListItemIcon,
+  Menu,
+  MenuItem,
+  Typography,
+  withStyles,
+  ListItemText
+} from "@material-ui/core";
```
Comment (dmbarry86) [inline src/UserMenu/UserMenu.js:14]: imports not in alphabetical order (not just the mui ones)

File: src/UserMenu/UserMenu.js [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+import { VpnKey, ExitToApp } from "@material-ui/icons";
+import {
+  usePopupState,
+  bindTrigger,
+  bindMenu
+} from "material-ui-popup-state/hooks";
+
+const useStyles = makeStyles(theme => ({
+  noFocus: {
```
Comment (dmbarry86) [inline src/UserMenu/UserMenu.js:23]: fields and sub-fields not in order

File: src/UserMenu/UserMenu.js [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+  username: PropTypes.string.isRequired
+};
+
+/**
+ * User avatar
+ *
+ * Provides custom styling and converts username to max 2 initials
+ */
+const UserAvatar = withStyles(theme => ({
+  root: {
```
Comment (dmbarry86) [inline src/UserMenu/UserMenu.js:124]: fields not in order

File: src/index.js [hunk 4]
```diff
@@ -1,5 +1,7 @@
 export { default as Copyright } from "./Copyright";
+export { default as ChangePasswordForm } from "./ChangePasswordForm";
```
Comment (dmbarry86) [inline src/index.js:2]: import should come before copyright

**Review by dmbarry86 — CHANGES_REQUESTED** [review]
Functionality looks good. Just the usual Matt Corner alphabet to sort 😉

File: src/UserMenu/UserMenu.js (hunk 1 above)
Comment (mattcorner) [reply src/UserMenu/UserMenu.js:14]: Have you not written a linter for this yet? 😂

**Comment by mattcorner** [comment]
I've turned on https://eslint.org/docs/rules/sort-imports to catch some of these alphabetical comments at source.

**Review by dmbarry86 — APPROVED** [review]

**Pull request merged**